src/Behaviours/Sluggable.php

```diff
@@ -63,27 +63,22 @@ public function generateTitleSlug(array $fields)
     /**
      * Because a title slug can be created from multiple sources (such as an article title, a category title.etc.),
      * this allows us to search out those fields from related objects and return the combined values.
-     *
-     * @param array $fields
-     * @return array
      */
-    public function getTitleFields(array $fields)
+    public function getTitleFields(array $fields): array
     {
-        $fields = array_map(function ($field) {
+        return array_map(function ($field) {
             if (Str::contains($field, '.')) {
                 return object_get($this, $field); // this acts as a delimiter, which we can replace with /
             } else {
                 return $this->{$field};
             }
         }, $fields);
-
-        return $fields;
     }
 
     /**
      * Generate the slug for the model based on the model's slug strategy.
      */
-    public function generateSlug()
+    public function generateSlug(): void
     {
         $strategy = $this->slugStrategy();
 
```

Comment:
Is there a pending implementation for `Uuid` ?
I see Ramsey's package being added in the `composer.json`

Reply:
There is not, now been removed.